daemon/info_unix.go

```diff
@@ -38,55 +37,44 @@ func (daemon *Daemon) fillPlatformInfo(v *types.Info, sysInfo *sysinfo.SysInfo)
 	v.Runtimes = daemon.configStore.GetAllRuntimes()
 	v.DefaultRuntime = daemon.configStore.GetDefaultRuntimeName()
 	v.InitBinary = daemon.configStore.GetInitPath()
+	v.RuncCommit.ID = "N/A"
+	v.ContainerdCommit.ID = "N/A"
+	v.InitCommit.ID = "N/A"
```

Comment:
Bit on the fence if we need to keep the `N/A` value on the server-side, or if we should move this to the cli. I'd love to move it there, but will have to think if there could be a breaking change because of that.

I might still do so in a follow-up though 😅